README.md

```diff
@@ -71,9 +71,6 @@ platforms:
     driver:
       image_urn: Canonical:UbuntuServer:14.04.4-LTS:latest
       vm_name: trusty-vm
-      vm_tags:
```

Comment:
This example seems like it should be added to the Windows VM example after line 100 since the description for the Windows VM example on line 92 includes a callout of the `vm_tags` parameter.

Reply:
Added!